iceoryx_posh/test/moduletests/test_mepoo_memory_manager.cpp

```diff
@@ -44,6 +45,22 @@ class MemoryManager_test : public Test
         free(rawMemory);
     };
 
+    using ChunkStore = std::vector<iox::mepoo::SharedChunk>;
```

Comment:
Why not `JunkStore`? :grin:

Reply:
I'll use `chunkStore` which is already used in another test

Reply:
I should read and think before responding 😁